### PR TITLE
add condtional render of 'Moved within rating year' based on 'Lived in property 1 July 2018'

### PR DESCRIPTION
**Pull request by mischa-s**
# What has changed and why?
Only render moved within rating year question if lived in property july 1 is false

# How to test this change

- [x] has tests
- [ ] at least one a reviewer ran the code

### CODE REVIEW

File: spec/features/admin/rebate_forms/edit_spec.rb [hunk 1]
```diff
@@ -28,6 +28,22 @@
...
+        choose('fields.lived_in_property_1_July', option: 'yes')
+        expect(page).to_not have_text('Moved within rating year')
+        choose('fields.lived_in_property_1_July', option: 'no')
+        expect(page).to have_text('Moved within rating year')
+        choose('fields.moved_within_rating_year', option: 'no')
+        expect(page).not_to have_text('Settlement date')
+        choose('fields.moved_within_rating_year', option: 'yes')
+        expect(page).to have_text('Settlement date')
+      end
+    end
```
Comment (CaraHill) [inline spec/features/admin/rebate_forms/edit_spec.rb]: I would rewrite it a little for clarity:
```
describe 'condtional render of fields' do
  it 'can hide moved within rating year and previous year information' do
    visit "/admin/rebate_forms/#{rebate_form.id}/edit"
    expect(page).to have_text('Customer details')
    expect(page).to have_text('Name')
    choose('fields.lived_in_property_1_July', option: 'yes')
    expect(page).to_not have_text('Moved within rating year')
    expect(page).to have_text('Settlement date')
    choose('fields.lived_in_property_1_July', option: 'no')
    expect(page).to have_text('Moved within rating year')
    expect(page).not_to have_text('Settlement date')
  end
end
```

Comment (mischa-s) [reply spec/features/admin/rebate_forms/edit_spec.rb]: hmmm this doesn't pass for me - tried to simplify it as much as possible does rspec pass for you with :arrow_double_up: ?

Comment (CaraHill) [reply spec/features/admin/rebate_forms/edit_spec.rb]: Give me one sec ...

Comment (CaraHill) [reply spec/features/admin/rebate_forms/edit_spec.rb]: Nope, you were totally right as is, I misread it. Sorry! Approving now.

**Pull request merged**